src/main/java/com/craftaro/epicfarming/database/migrations/_1_InitialMigration.java

```diff
@@ -38,7 +38,7 @@ public void migrate(Connection connection, String tablePrefix) throws SQLExcepti
         try (Statement statement = connection.createStatement()) {
             statement.execute("CREATE TABLE IF NOT EXISTS " + tablePrefix + "items (" +
                     "farm_id INTEGER NOT NULL, " +
-                    "item VARBINARY(999) NOT NULL" +
+                    "item TEXT NOT NULL" +
```

Comment:
Theses migrations only run once and *usually* are not intended to be edited afterwards (sometimes there have to).

Should this type change also be applied to existing installations? In that case there should be a new Migration-Class in the same directory that runs an `ALTER TABLE`. Often the previous migrations are not touched, so the `ALTER` query makes sense when reading the migration steps, as there is usually no downside for existing or fresh installs in that case.

Reply:
I kept this as TEXT because it wouldn't have worked the other way and any existing data would be corrupted anyway so this seems to make sense but can be changed if you'd like